Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:buster
+FROM arm64v8/debian:buster
```

Comment:
Do we really need to specify platform-specific docker image in the common (builder) dockerfile?)
I've tried these changes, and for BRCM (x86 platform) i gave up compliing image after ~45 minutes
(Without this line that specifies platform in the docker image it took me ~15-20 minutes clean build till completion)
I think the platform specific img version in my case falls back to using qemu and hence slows down the compile process on the host OS;

Root dockerfile is the <build enviroment> image... do you really need to use arm64 here? Is it expected that you'd be generating an image / deb package on the ARM system?

Reply:
<Discussed offline through slack, ignore this comment>